src/matchers/anyString.js

```diff
@@ -0,0 +1,20 @@
+import Matcher from './matcher';
+class AnyString extends Matcher {
+  constructor(stringToMatch){
+    super();
+    this.value = stringToMatch;
+  }
+  matches(){
+    if (!this.value) {
+      return false;
+    }
+    if (typeof this.value === 'string' || this.value instanceof String)
```

Comment:
Exactly the logic you would need to do in Java to avoid null pointer exceptions, but in JS if `this.value` is undefined `typeof this.value === 'string'` or `this.value instanceof String` will just return false, it won't throw an error.